### DISCUSSION TITLE
Welcome to ScintillaNET Discussions!

### DISCUSSION THREAD

**Discussion by VPKSoft** [desjarlais/Scintilla.NET#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cgm-wmaisenhelder** [comment]
I'm currently using Scintilla.NET version 5.3.2.9.  I'm getting an error when attempting to open any form that contains a Scintilla control in the Visual Studio IDE.  The forms do open correctly when running the application, I just can't open them in the IDE.

The forms were originally created using Jacob Slusser's ScintillaNET 2.5.2, and were recently updated to the VPK version mentioned above.

The error says "Unable to locate the Scintilla.NET satellite assemblies : directory 'C\Program Files (x86)\Microsoft Visual Studio\2019\Professional\Common7\IDE\packages\Scintilla.NET.5.3.2.9\build\x86' not found".

There is no "packages" folder in the "IDE" folder.  The Scintilla package does exist in my "C:\Users\<me>\.nuget\packages" folder.

I know this may be more of a Visual Studio or nuget configuration issue, but I've searched a solution but haven't found anything.  I'm hoping someone with a little more knowledge of how this is supposed to work can help.

**Comment by desjarlais** [comment]
@cgm-wmaisenhelder : Version 5.3.2.9 is the previous version of the Scintilla.NET library.  That was maintained by VPKSoft and I have since taken over for him, but to update the library on nuget, I needed to create a new library under my nuget account.  You can try the "Scintilla5" nuget library and see if that resolves the issue.

https://www.nuget.org/packages/Scintilla5.NET

If you still see the problem, create an "Issue" on this github repro and I'll try to figure out what is wrong.

**Comment by cgm-wmaisenhelder** [comment]
Thanks!  The Scintilla5.NET package fixed my issue with adding the component to the VS toolbox, and also displaying the component on the form designer.

Secondary question, is the new package compatible with the AutoComplete-Scintilla.NET package?

Best regards,

Welton Maisenhelder v3.63
Senior Software Developer

BEST IN KLAS WINNER<https://www.cgm.com/usa_en/articles/press-releases/compugroup-medicals-aria-rcm-services-ranked-1.html> | AMBULATORY RCM SERVICES (EHR-ASSOCIATED)

***@***.***

T +1 (800) 359-0911 x 1033
F +1 (410) 517-0238

***@***.******@***.***>
www.cgm.com/us<http://www.cgm.com/us>

Synchronizing Healthcare
Before printing, think about the environment.
CompuGroup Medical Confidentiality Notice: The information contained in this message, including any attachments, is confidential and may contain Protected Health Information or may be otherwise legally privileged. This message is intended solely for the addressee(s). If you are not the intended recipient, be advised that any unauthorized use, disclosure, copying, distribution or the taking of any action in reliance on the contents of this information is strictly prohibited, may be unlawful and may subject you to liability. If you have received this message in error, please contact the sender by return phone, email or fax and destroy all copies of the original message.

**Comment by desjarlais** [comment]
I don't have ownership of AutoComplete-Scintilla.NET, it looks like it was archived with an older version, so it will not be compatible with this new package.

**Comment by cgm-wmaisenhelder** [comment]
OK, thanks for the info.

Best regards,

Welton Maisenhelder v3.63
Senior Software Developer

BEST IN KLAS WINNER<https://www.cgm.com/usa_en/articles/press-releases/compugroup-medicals-aria-rcm-services-ranked-1.html> | AMBULATORY RCM SERVICES (EHR-ASSOCIATED)

***@***.***

T +1 (800) 359-0911 x 1033
F +1 (410) 517-0238

***@***.******@***.***>
www.cgm.com/us<http://www.cgm.com/us>

Synchronizing Healthcare
Before printing, think about the environment.
CompuGroup Medical Confidentiality Notice: The information contained in this message, including any attachments, is confidential and may contain Protected Health Information or may be otherwise legally privileged. This message is intended solely for the addressee(s). If you are not the intended recipient, be advised that any unauthorized use, disclosure, copying, distribution or the taking of any action in reliance on the contents of this information is strictly prohibited, may be unlawful and may subject you to liability. If you have received this message in error, please contact the sender by return phone, email or fax and destroy all copies of the original message.
